Java/src/main/java/airport/Airport.java

```diff
@@ -27,11 +29,8 @@ public List<MilitaryPlane> getMilitaryPlanes() {
         for (Plane plane : planes) {
             if (plane instanceof MilitaryPlane) {
                 militaryPlanes.add((MilitaryPlane) plane);
-            } //if
-            else {
-
-            } // else
-        } //for
+            }
+        }
```

Comment:
on the lines 40, 57 and 69 for-loop can be replaced with foreach

Reply:
Done I think)